src/components/grid-aware/BlockQuoteBlock/BlockQuoteBlock.js

```diff
@@ -0,0 +1,28 @@
+import PropTypes from "prop-types";
+import React from "react";
+import s from "./BlockQuoteBlock.module.css";
+
+const BlockQuoteBlock = ({ quote, author }) => {
+  const GridAreaMiddle = () => (
+    <div className={s.gridAreaMiddle}>
+      <div className={s.quote}>{quote}</div>
+      <br />
+      <div className={s.author}>{author}</div>
+    </div>
+  );
+
+  return (
+    <div className={s.bleedWrapper}>
+      <div className={s.bleedMainContent}>
+        <GridAreaMiddle />
+      </div>
+    </div>
+  );
+};
+
+BlockQuoteBlock.propTypes = {
+  quote: PropTypes.string.isRequired,
+  author: PropTypes.string.isRequired,
+};
```

Comment:
I think we should make this component include the actual carousel, so that this component takes in an array of quotes+attributions. That way, when you want to use it with multiple quotes, you don't have to create the carousel yourself.

Reply:
I added an array of objects with key quote and value attribution. Not sure if that's how you envisioned it. Please let me know.

Reply:
👍 Created the blockQuoteblock component to have propTypes of an array that consists of keys "quote" and "attribution", which consists of values with propTypes of strings.